lte/gateway/Vagrantfile

```diff
@@ -28,7 +28,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define :magma, primary: true do |magma|
     magma.vm.box = "magmacore/magma_dev"
-    magma.vm.box_version = "1.2.20221012"
```

Comment:
Same as in the workflows: I think it would be reasonable to change the version to `1.3` since there are actual changes to the base image.